app/assets/stylesheets/publisher.scss

```diff
@@ -98,6 +98,12 @@
         line-height: $line-height-base !important;
       }
 
+      .twitter-typeahead {
+        position: absolute !important;
+
+        .tt-menu { margin-top: 0; }
```

Comment:
I think the reasons given [here](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#selectordepth) are important enough to keep this rule.